using-timescaledb/update-db.md

```diff
@@ -34,6 +34,21 @@ the load of a previous DB version on session startup.
 
 <!-- -->
 
+>:WARNING: If you are upgrading from a version before 0.11.0 make sure your 
+root table does not contain data otherwise the update will fail.
+Data can be migrated as follows:
+```sql
+BEGIN;
+SET timescaledb.restoring = 'OFF';
```

Comment:
case does not matter, but we are not using it consistently then, cause in the code its all uppercase

https://github.com/timescale/timescaledb/blob/master/src/hypertable.c#L1168

but if show setting in postgres it will always be lowercase on/off

Reply:
I've changed it to lowercase and will do PR to change the hints in timescaledb aswell.